elections57_results.py

```diff
@@ -220,6 +228,23 @@ def get_class(v):
         for c in sorted(classes):
             votes_by_class.append(votes[c])
         print '{},{}'.format(y, ','.join([str(x) for x in votes_by_class]))
+elif cmdline_args.operation == 'raw_data':
+    SALT = input("Enter salt: ") # Or any other method to get SALT known to organizers
+    OUTPUT = 'raw_results_anonymized.csv'
+
+    def hash(code):
+        return hashlib.sha256(code + SALT).hexdigest()
```

Comment:
I _think_ this is secure enough, but I'm _not_ 100% sure that this is safe if somebody really bothers to precompute a "rainbow table" with enough hashes and all voters names (see below). I'd use bcrypt just to be totally safe.